python/torch_mlir_e2e_test/test_suite/basic.py

```diff
@@ -129,6 +129,32 @@ def MmDagModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(4, 4), tu.rand(4, 4))
 
 
+# ==============================================================================
+
+class ContainsIntList(torch.nn.Module):
+    def __init__(self):
+        self.l = [1,2,3]
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1], torch.float32, True)
+    ])
+    def forward(self,key):
+        
+        return torch.ops.aten.__contains__(self.l, key)
+
+
+@register_test_case(module_factory=lambda: ContainsIntList())
+def ContainsIntList_True(module, tu: TestUtils):
+    module.forward(3)
```

Comment:
This is not right. The inputs should all be tensors. This is an assumption made by the `annotate_args` above